Feature #11763: Add total downloads count to FileList

SashaMcr opened the pull request:
This commit adds the total downloads count to the FileList and FolderItemList. It creates a DownloadsColumn class specifically for a new column in the file search menu to display total downloads. This update provides a clearer and more detailed display for the user.

*Check these before submitting new pull requests*

- [ ] I read the __guidelines for contributing__ linked above. My pull request conforms to the coding style guidelines described within.

If this condition is met, feel free to delete this whole message and to submit your pull request.

Thank you, your help is really appreciated!

aembler commented:
Looks like this needs to have conflicts fixed now that it's being adopted in the 9.3.x branch.

Commit:
Feature concretecms#11763: Add total downloads count to FileList
This commit adds the total downloads count to the FileList and FolderItemList. It creates a DownloadsColumn class specifically for a new column in the file search menu to display total downloads. This update provides a clearer and more detailed display for the user.
